src/commands/endpoints/create.ts

```diff
@@ -18,12 +18,13 @@ export default class EndpointsCreate extends BaseCommand {
   }
 
   static examples = [
-    '$ heroku endpoints:create',
+    '$ heroku endpoints:create postgresql-sushi-12345 --account-ids 12345:user/abc --app trusted-vpc-endpoint-demo',
+    '$ heroku endpoints:create postgresql-sushi-12345 --account-ids 12345:user/abc 45678:user/xyz --app trusted-vpc-endpoint-demo',
   ]
 
   async run() {
     const {args, flags} = this.parse(EndpointsCreate)
-    const account_ids = flags['account-ids'].split(',').map((account: any) => account.trim())
+    const account_ids = flags['account-ids'].split(' ').map((account: any) => account.trim())
```

Comment:
NBD, but I think you could add a [custom ARN list flag type](https://oclif.io/docs/flags#custom-flags) that takes a whatever-delimited string as an argument and returns an array of ARNs and share that code across commands.